class-instant-articles-post.php

```diff
@@ -296,7 +296,7 @@ protected function _get_the_content() {
 
 		// If we’re not it the loop or otherwise properly setup.
 		$reset_postdata = false;
-		if ( $this->_post->ID !== $post->ID ) {
+		if ( empty( $post ) || $this->_post->ID !== $post->ID ) {
```

Comment:
Adding this plugin to a VIP site caused issues when running unit tests.
On Sat, May 28, 2016 at 3:10 AM Stuart Shields <notifications@github.com>
wrote:

> In class-instant-articles-post.php
> <https://github.com/Automattic/facebook-instant-articles-wp/pull/301#discussion_r64985112>
> :
>
> > @@ -296,7 +296,7 @@ protected function _get_the_content() {
> >
> >  		// If we’re not it the loop or otherwise properly setup.
> >  		$reset_postdata = false;
> > -		if ( $this->_post->ID !== $post->ID ) {
> > +		if ( empty( $post ) || $this->_post->ID !== $post->ID ) {
>
> @srtfisher <https://github.com/srtfisher> What instance did you come
> across to need this?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Automattic/facebook-instant-articles-wp/pull/301/files/3a281fca4daef5af11ce01cd7446c51a4f191696#r64985112>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAVJH1DrYWJBWb-aicXnnO26Uy0OCbU1ks5qF-p1gaJpZM4Io2Km>
> .
>